scripts/apitypings/main.go

```diff
@@ -867,10 +867,14 @@ func (g *Generator) typescriptType(ty types.Type) (TypescriptType, error) {
 			return TypescriptType{ValueType: "Record<string, string>"}, nil
 		case "github.com/coder/coder/v2/cli/clibase.URL":
 			return TypescriptType{ValueType: "string"}, nil
-		// XXX: For some reason, the type generator generates this as `any`
+		// XXX: For some reason, the type generator generates these as `any`
 		//      so explicitly specifying the correct generic TS type.
 		case "github.com/coder/coder/v2/codersdk.RegionsResponse[github.com/coder/coder/v2/codersdk.WorkspaceProxy]":
 			return TypescriptType{ValueType: "RegionsResponse<WorkspaceProxy>"}, nil
+		case "github.com/coder/coder/v2/coderd/healthcheck/health.Message":
+			return TypescriptType{ValueType: "HealthMessage"}, nil
+		case "github.com/coder/coder/v2/coderd/healthcheck/health.Severity":
```

Comment:
Not sure why it generated a stub for `health.Severity` is partially correct, but as long as it works, I don't have follow up questions :)

Reply:
:+1:  Still want to figure out where the `null` slice is sneaking in...